soul-plugin/soul-plugin-ratelimiter/src/main/java/org/dromara/soul/plugin/ratelimiter/executor/RedisRateLimiter.java

```diff
@@ -41,15 +42,22 @@
 @Slf4j
 public class RedisRateLimiter {
     
+    public static final String RATE_LIMITER_ALGORITHM = "rateLimiterAlgorithm";
+    
+    public static final String KEYS = "keys";
+
+    public static final String SCRIPT_ARGS = "scriptArgs";
```

Comment:
I think the key of the attribute here maybe  need a prefix like soul or ratelimiter to make the meaning clearer.